### PR TITLE
Remove BatchNorm1d function

### PR DESCRIPTION
**Pull request by dcu**
It can be done with just one function for both 1d and 2d batch norms.

### CODE REVIEW

**Comment by chewxy** [comment]
I love this. Smaller API surface. What a brilliant thing
